QUARKUS-1325.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-1325 - Helm chart support for Quarkus
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1325
+
+This test plan cover Quarkus - Helm integration
+
+## Scope of the testing
```

Comment:
Openshift => OpenShift  (4 occurrences)
Quarkus Test-suite => Quarkus test-suite (1 occurrence)

Reply:
This is still not fixed!